source/2019-12-20-the-ember-times-issue-129.md

```diff
@@ -1,37 +1,33 @@
 ---
 title: The Ember Times - Issue No. 129
-author: Chris Ng, Amy Lam, Isaac Lee, the crowd
+author: Chris Ng, Amy Lam, Isaac Lee, Godfrey Chan, Jessica Jordan
```

Comment:
Not sure if I deserve the credit here but sure! 😅

Reply:
Of course! :-)